### ISSUE TITLE
Sumatra locks folder with opened pdfs

### ISSUE THREAD

**Issue by PF4Public** [sumatrapdfreader/sumatrapdf#556]
It is a long-persisting bug, but hard to recreate. It might have some relation to #536, but slightly different in behaviour. I remember it from the time I started using SumatraPDF.
The problem is that sometimes SumatraPDF locks upper level folder as in mentioned bug, but for me, this folder is locked completely: that is I cannot navigate to it nor save files there. Should I close files in SumatraPDF, without exiting it (contrary to named bug) that folder is immediately deleted with all its contents. This might be because the folder in question is locked both by SumatraPDF and Opera browser since it is "temporary_downloads", but closing file in SumatraPDF the latter releases lock (contrary to mentioned bug) and the folder is possibly removed by Opera. In the mean time, until I close files in SumatraPDF, that folder is completely inoperable, so that I cannot download any more files to it by Opera. That is especially frustrating if I have a bunch of pdfs open and need to open another one. To do so I have to close all of them and reopen again.

**Comment by kjk** [comment]
I'll assume that just as #536, it was fixed by https://github.com/sumatrapdfreader/sumatrapdf/commit/c863232e153ca3ba62f91853ee4c0631a51ac37b

**Comment by PF4Public** [comment]
Hi, Krzysztof
Recently I have encountered this bug once again and, remembering that you've closed this issue, tried to install prerelease version, but this bug is still unsolved. It might be related, but not the same bug as the one mentioned.
Through trial and error I came to the way to reproduce this bug.
Try as folows:
1. Create any folder. Lets say "Folder"
2. Put any .pdf file in it
3. Now via drag-drop create a copy, with windows giving it automatic name
4. Open that .pdf from the folder, created at 3. (not the original one)
5. Keep SumatraPDF running with this file opened, shift-delete that folder, created at 3. (not the original one)
6. Windows obeys and completely deletes that folder along with files within even if one is opened in SumatraPDF
7. Try the same as in 3. to create another copy with the same name.
Expected outcome of 7. is that a new copy is created, but in reality you get windows warning about insufficient rights to perform the operation. Inspecting SumatraPDF in Process Explorer reveals the fact that it still holds active handle for the deleted folder and as such it might prevent creating another one in place. 
Is it a bug in SumatraPDF or is it absolutely normal behavior?
